lib/pf/ConfigStore/Switch.pm

```diff
@@ -27,6 +27,21 @@ with 'pf::ConfigStore::Hierarchy';
 
 sub configFile { $switches_config_file }
 
+=head2 _buildCachedConfig
+
+Build the pf::config::cached object
+
+=cut
+
+sub _buildCachedConfig {
```

Comment:
Can you document shortly why this has to be redefined in this ConfigStore instead of using the parent one